documentation/Installation.md

```diff
@@ -20,7 +20,7 @@ Saffire. Once the box has started, issue the following:
 
     vagrant ssh
     cd /vagrant
-    sh build.sh
+    sudo ./build.sh -i
```

Comment:
Ok.. let's remove the "install" text and use `./build/release/bin/saffire` for now.. We could add a separate section about installing.